spec.html

```diff
@@ -33282,7 +33566,12 @@ <h1>
           <dl class="header">
           </dl>
           <emu-alg>
-            1. Let _offset_ be LocalTZA(_tv_, *true*).
+            1. Let _localTimeZone_ be DefaultTimeZone().
+            1. If IsTimeZoneOffsetString(_localTimeZone_) is *true*, then
+              1. Let _offsetNs_ be ParseTimeZoneOffsetString(_localTimeZone_).
+            1. Else,
+              1. Let _offsetNs_ be GetNamedTimeZoneOffsetNanoseconds(_localTimeZone_, ℤ(ℝ(_t_) &times; 10<sup>6</sup>)).
```

Comment:
```suggestion
              1. Let _offsetNs_ be GetNamedTimeZoneOffsetNanoseconds(_localTimeZone_, ℤ(ℝ(_tv_) &times; 10<sup>6</sup>)).
```
Moreover, this formula won't work for a lot of non-NaN Numbers, so you should probably change the type of `_tv_` to `an integral Number`.

And since ToDateString calls this op, you should probably change the type of ToDateString's `_tv_` to `an integral Number or *NaN*`.